app/components/object-inspector/properties-base.js

```diff
@@ -5,6 +5,12 @@ export default Component.extend({
   tagName: '',
 
   sendToConsole: action(function ({ name }) {
+    if (name === '...') {
```

Comment:
What is the idea behind this `if`? Is it just to avoid sending the `property: name` from the code below? If so, perhaps we could conditionally add the `property` prop instead of duplicating the `this.port.send('objectInspector:sendToConsole',` call?